recipes/pycovjson/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pycovjson" %}
+{% set version = "0.3.9" %}
+{% set sha256 = "f1482516307189f8b7b8b0f3eeea645923298cea2bfb67f43f0d5e29fa2cc504" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - xarray
+    - numpy
+    - pandas
+    - pymongo
+
+test:
+  imports:
+    - pycovjson
+
+about:
+  home: https://github.com/Reading-eScience-Centre/pycovjson
+  license: BSD 3-clause
+  license_family: BSD
+  license_file: LICENSE
```

Comment:
It looks like the license file is not distributed with the source. Please raise and issue upstream and copy the link to the issue as a comment in the recipe. Then you can manually include it:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
ack, done

ISSUE: https://github.com/Reading-eScience-Centre/pycovjson/issues/38
PULL REQUEST: https://github.com/Reading-eScience-Centre/pycovjson/pull/39